.github/workflows/build-sphinx.yml

```diff
@@ -1,26 +1,13 @@
 name: User's Guide
 on:
-    schedule:
-        # Rebuild daily, just after (30 minutes after) any tags were updated.
-        # I'd prefer to rebuild in response to added tags, but that's
-        # much harder to make happen on github actions.
-        - cron: '30 0 * * *'
-
     push:  # Build the docs for any branch (we only push to GCP when building from main)
-        branches:
-            - '*'
-
-    create:  # Any time a tag is added, rebuild the docs.
-        branches:
-            - main

```

Comment:
I don't think this was having any effect because it's not listed in the [workflow syntax reference](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore).